index.html

```diff
@@ -2,9 +2,87 @@
   </head>
   <body>
+    <div class="header">
+      <div class="banner">
+              <div class="navbar">
+        <img class="navbar__logo" src="http://placecage.com/50/50" alt="nav logo">
+        <div class="links">
+          <h3 class="link__item">About</h3>
+          <h3 class="link__item">Search</h3>
+        </div>
+              </div>
+              <div class="hero">
+        <h1 class="title">Surf & Paddle Co.</h1>
+        <p class="subtitle">An Ironyard Academy Project Est. Fall 2013</p>
+      </div>
+      </div>
+    </div>
+
+  <div class="site_main">
+    <div class="left-article">
+
+      <div class="article__heading">A Fantasting Engaging Heading Goes Here</div>
+      <div class="article__info">
+        <img class="article__author-avatar" src="http://www.placecage.com/30/30" alt="avatar">
+        <div class="author__name-left">Annie Marie</div>
+        <div class="article__publish-date">on Monday Oct. 7th 9:22pm</div>
+      </div>
+      <div class="article__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </div>
+      <div class="article__quote"> WOAH, THAT'S RADICAL!</div>
+      <div class="article__text2">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus veniam aut voluptatem distinctio eaque, esse ex fugit rem laborum quo. Quas voluptatibus nobis laboriosam harum fugit magnam odio officiis illum!</div>
+      </div>
+
+    <div class="right-article">
+        <div class="author__info">
+          <img class="author__image-right" src="http://www.placecage.com/50/50" alt="avatar">
+                <div class="author__name-right">Annie Marie</div>
+                <div class="author__social">
+          <a href="http://www.twitter.com/steftheo11">Twitter</a>
+          <a href="http://www.facebook.com">Facebook</a>
+          <a href="http://www.instagram.com">Instagram</a>
+                </div>
+        </div>
+      <img src="http://www.placecage.com/200/200" alt="Ad Image">
+      <a class="about" href="http://www.google.com">About this Ad</a>
+      <h3 class="sidebar-section-heading">Sidebar Section Heading</h3>
+      <p class="sidebar__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Recusandae, sequi nemo dicta praesentium similique ipsa fuga perspiciatis quas omnis, distinctio totam minus iure doloribus delectus id! Quaerat numquam nobis reprehenderit.</p>
+    </div>
+
+
+  </div>
+
+  <div class="main__footer">
+    <div class="footer">
+      <h1 class="title__footer">Most Popular Posts</h1>
+      <div class="images__footer">
```

Comment:
Each title and image in the footer should be grouped together. This will make things easier to position and is more understandable from an "outline" perspective